docs/tools/vdb_table/data/mongo.json

```diff
@@ -144,7 +144,7 @@
     "comment": ""
   },
   "vector_dims": {
-    "value": 0,
+    "value": 4096,
```

Comment:
Hi @hweller1, thanks for updating here. Could you provide a source url where this number is documented?

Reply:
please update the PR when this is updated: https://www.mongodb.com/docs/atlas/atlas-search/field-types/knn-vector/#configure-fts-field-type-field-properties. Thanks

Reply:
@dhruv-anand-aintech it's updated now!